NuGet.Config

```diff
@@ -10,6 +10,7 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="azure-sdk-tools@DevOps" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-tools/nuget/v3/index.json" />
```

Comment:
Is this needed yet? Can we eliminate the other feed? I'm trying to keep us down to a minimal set of feeds.

Reply:
I removed the new feed and added it directly as the source in the Update-Snippet Script.